mimic/model/mailgun_objects.py

```diff
@@ -118,5 +118,5 @@ def filter_message_by_to_address(self, to_address):
         Retrieve a :obj:`Message` object by its `to` address.
         """
         for each_msg in self.message_store:
-            if each_msg.to == to_address[0]:
+            if each_msg.to == to_address:
```

Comment:
Someday everything will be Python 3+ and there will be types and things
like this will be caught at the first code review.

On Tuesday, October 27, 2015, Glyph <notifications@github.com> wrote:

> In mimic/model/mailgun_objects.py
> <https://github.com/rackerlabs/mimic/pull/471#discussion_r43209529>:
>
> > @@ -118,5 +118,5 @@ def filter_message_by_to_address(self, to_address):
> >          Retrieve a :obj:`Message` object by its `to` address.
> >          """
> >          for each_msg in self.message_store:
> > -            if each_msg.to == to_address[0]:
> > +            if each_msg.to == to_address:
>
> Also: thanks for having a look at this :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rackerlabs/mimic/pull/471/files#r43209529>.
>